data/json/npcs/old_guard/NPC_Old_Guard_FOB_officer.json

```diff
@@ -0,0 +1,91 @@
+[
+  {
+    "type": "npc",
+    "id": "old_guard_fob_officer",
+    "//": "Generic guard for the old guard.",
+    "name_suffix": "Officer",
+    "name_unique": "Duhring",
+    "class": "NC_OG_OFFICER",
+    "gender": "male",
+    "attitude": 0,
+    "mission": 8,
+    "chat": "TALK_OLD_GUARD_FOB_OFFICER",
+    "faction": "old_guard"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_OG_OFFICER",
+    "name": { "str": "Officer" },
+    "job_description": "I'm an officer for FOB Arabella",
+    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "common": false,
+    "bonus_int": { "one_in": 4 },
+    "bonus_per": { "one_in": 4 },
+    "worn_override": "NC_OG_OFFICER_worn",
+    "carry_override": "NC_OG_OFFICER_misc",
+    "proficiencies": [ "prof_gunsmithing_basic" ],
+    "skills": [
+      { "skill": "speech", "bonus": { "rng": [ 3, 5 ] } },
+      { "skill": "barter", "bonus": { "rng": [ 1, 3 ] } },
+      { "skill": "gun", "bonus": { "rng": [ 3, 5 ] } },
+      { "skill": "pistol", "bonus": { "rng": [ 3, 5 ] } },
+      { "skill": "rifle", "bonus": { "rng": [ 3, 5 ] } },
+      { "skill": "driving", "bonus": { "rng": [ 1, 3 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_OG_OFFICER_worn",
+    "subtype": "collection",
+    "items": [
+      { "item": "socks", "prob": 30 },
+      { "item": "briefs" },
+      { "item": "pants_army" },
+      { "item": "jacket_army" },
+      { "item": "dress_shoes" },
+      { "item": "gloves_leather", "prob": 80 },
+      { "item": "webbing_belt", "prob": 80 },
+      { "item": "m18", "ammo-item": "9mmP2", "container-item": "holster", "charges": 16 },
+      { "distribution": [ { "item": "gold_watch", "prob": 30 }, { "item": "sf_watch", "prob": 70 } ] },
+      { "item": "collarpin" },
+      { "item": "diamond_silver_cufflinks" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_OG_OFFICER_misc",
+    "subtype": "collection",
+    "items": [
+      { "item": "two_way_radio" },
+      { "item": "p320mag_17rd_9x19mm", "prob": 45 },
+      { "item": "pencil", "count": 2 },
+      { "item": "pen" },
+      { "item": "rag" },
+      { "item": "red_pen" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_OLD_GUARD_FOB_OFFICER",
+    "dynamic_line": [ "I don't have any use for you right now.  You will be informed if I need you, soldier." ],
```

Comment:
Is the player supposed to always be a member of the Old Guard at the point when they can first interact with this NPC?

Reply:
I believe so. The base is established as a result of the PC performing a quest for the Old Guard (clearing out the location), so they shouldn't set up shop until the PC has done that. However, it's not impossible that there is (or will be) a way to fall out with the Old Guard so you no longer would be considered a member, but possibly not (yet) an enemy. I'd expect such an untrustworthy character wouldn't be "recruited" again, though.

Reply:
> the player supposed to always be a member of the Old Guard at the point when they can first interact with this NPC?
Nope. No real easy ways of doing this if its desirable even but I think that around the midpoint of OG tree you will start getting penalized for not doing some tasks in a reasonable timeframe to represent a greater trust placed in you and higher importance of of work given to the player.